helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml

```diff
@@ -12,8 +12,8 @@ spec:
   strategy: production
 
   query:
-    {{- if and .Values.jaeger.hub .Values.jaeger.tag }}
-    image: {{ .Values.jaeger.hub }}/jaeger-query:{{ .Values.jaeger.tag }}
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag .Values.jaeger.jaegerQueryImage }}
```

Comment:
nit: `.Values.jaeger.jaegerQueryImage` -> `.Values.jaeger.queryImage`?
and similar for the other image names below.

Reply:
Should be renamed now